### ISSUE TITLE
no live server option in right click context menu.

### ISSUE THREAD

**Issue by k1nghat** [ritwickdey/vscode-live-server#215]
## I'm submitting a...

```html
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Other:
```

## Current behavior
when i right click on .html file i see no options for live server.

![image](https://user-images.githubusercontent.com/16073360/46515479-aeff9580-c829-11e8-8da5-cc16186f34de.png)

## Expected behavior
there should be 'open with live server' option.

## Environment

```html
Browser:

- [x] Chrome (desktop) version 69
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] IE version XX
- [ ] Edge version XX

For Tooling issues:

- Live Server: 5.1.1
- Platform: linux(kubuntu)
- Visual Studio Code: 1.27.2
```

**Comment by KshitijV97** [comment]
Same, Any solution?

**Comment by Markxuxiao** [comment]
![image](https://user-images.githubusercontent.com/9879038/54742441-6a5f8f00-4bfc-11e9-8c28-c881837e121f.png)

**Comment by k1nghat** [comment]
this is not an issue for me any longer. can be closed.

**Comment by BOCbMOU** [comment]
Well, this is an issue for me. I use VS Code on WSL and don't have this option on right click. On non WSL version I have it. Reinstalled extension 2-3 times and nothing has changed.

**Comment by dcbentley** [comment]
This is also something I'm experiencing. The only way I can use it is by clicking the Live button on the blue bar at the bottom. (there are also no shortcut keys combos that I can press to launch it).

VS Code 1.46.1
Electron 7.3.1
Chrome 78.0.3904.130
Node.js 12.8.1
V8 7.8.279.23-electron.0
OS Darwin x64 19.5.0 (Catalina v10.15.5)

**Comment by prodevengineer** [comment]
Yes, just came across this problem this afternoon. I have to type Liveserver on CTRL SHIFT P and search for it everytime.
But I like the suggestion to click on the blue bar. Quicker though.
Thanks @dcbentley

**Comment by sandeep-v1404** [comment]
Once Try this Fix Guys..

Go to **File -> Preferences -> Keyboard Shortcuts** in the menu of VSCodeEditor.

Search for "live" . You will get -> Live Server:Open with Live Server. Click on it. Set a Unique Key Binding (Mine is "Alt+m") . Press Enter . 
_Now Check the Right Click Context Menu_ 
### 
I don't no but this may work .... ### Just Try.   :)

**Comment by vitordutra** [comment]
I'm also experiencing this on my Linux Mint 19.3 Cinammon. Right click option was working without any issues before. Maybe it was related to some update? VSCode seems a little buggy lately for me.

**Comment by HelaBela** [comment]
What worked for me: put the html in a folder and open folder form VS. Then right click on the html and the option is there

**Comment by longmarch1** [comment]
What I found is that, you are right clicking the file in "Open Editors". (As I am in the same shoes.)

Instead, right click on the file in the lower panel. "Open with Live Server" will show.

For an opened file, there is a blue bar at the bottom, where you can click "Go Live" to see the live view. 

Hope it helps.

**Comment by Taxi000** [comment]
> Instead, right click on the file in the lower panel. "Open with Live Server" will show.

Dude, you are my hero!

**Comment by prachi24cross7** [comment]
In github codespaces, 

Go to File -> Preferences -> Extensions and search 'live' -> you will get the option 'live server' -> install it -> now you have the option

**Comment by Salman512AA** [comment]
simplest solution click extension button on left side and download live server

**Comment by chimdi007** [comment]
I have installed live server and enabled it on my vs studio code, but when I right cick on the html file, I don't see the 'open with live server' option

**Comment by hujimongola** [comment]
i m having same problems same as you. if you find sol so give me tooo

> I have installed live server and enabled it on my vs studio code, but when I right cick on the html file, I don't see the 'open with live server' option
